.gitlab-ci.yml

```diff
@@ -12,6 +13,7 @@ variables:
   DOCKER_TLS_CERTDIR: "/certs"
   DOCKER_BUILDKIT: 1
 
+# this defines two variables .default_rules.default and .default_rules.release which are True when the if is satisfied
```

Comment:
```suggestion
This key define variables which later use in `!reference` tag in `rules`. 
# Ref https://docs.gitlab.com/ee/ci/jobs/index.html#hide-jobs 
# and https://docs.gitlab.com/ee/ci/yaml/yaml_optimization.html#reference-tags
```

Reply:
OK. Please change "which later" to "which are later used" . (avoid a verb w/o a subject)

Reply:
I do not seem able to edit your commit

Reply:
Done